plugins/dbnd-docker/src/dbnd_docker/kubernetes/kubernetes_engine_config.py

```diff
@@ -92,15 +92,14 @@ class KubernetesEngineConfig(ContainerEngineConfig):
     requests = parameter.none()[Dict]
     limits = parameter.none()[Dict]
 
-    pod_exit_code_to_retry_count = parameter(empty_default=True).help(
-        "Mapping between pod exit code to amount of pod retry attempts"
-    )[Dict]
-    pod_retry_delay = parameter.help(
-        "The delay between each pod retry attempt in time delta format. 1m, 5s, 1h, etc."
-    )[datetime.timedelta]
-    retry_on_image_pull_error_count = parameter.help(
-        "Describes the amount of retry attempts when a pod fails with " "'ErrImagePull'"
-    ).default(0)[int]
+    pod_error_cfg = parameter(
```

Comment:
Let's make it into an object, and have the magic `exit_code` and `retry_count` strings as consts within that object